inc/functions_upload.php

```diff
@@ -313,6 +313,9 @@ function upload_avatar($avatar=array(), $uid=0)
 		case "image/x-png":
 			$img_type = 3;
 			break;
+		case "image/bmp":
```

Comment:
Only the most common ones should be there: image/bmp, image/x-bmp, image/x-bitmap